R/durations.r

```diff
@@ -101,6 +101,7 @@ format.Duration <- function(x, ...) {
   out <- vector("character", length(x@.Data))
   nnas <- !is.na(x@.Data)
   out[nnas] <- compute_estimate(abs(x@.Data[nnas]))
+  out[nnas] <- ifelse(x@.Data[nnas] < 0, paste0("-", out[nnas]), out[nnas])
```

Comment:
I think this negative sign should be handled in `compute_estimate` instead and that `abs` removed.